aospy/model.py

```diff
@@ -235,21 +235,25 @@ def _set_mult_grid_attr(self):
                 grid_attr = _get_grid_attr(grid_objs, name)
                 if grid_attr is not None:
                     # Rename coordinates to aospy's internal names.
+                    TIME_STR = internal_names.TIME_STR
                     renamed_attr = _rename_coords(grid_attr)
+                    if ((TIME_STR not in renamed_attr.dims) and (TIME_STR in renamed_attr)):
+                        renamed_attr = renamed_attr.drop(TIME_STR)
                     # Hack: MERRA data ending up with time-defined lat- and
```

Comment:
Actually, can we get rid of this MERRA-related logic entirely?  If/when it comes up again, we can do more cleanly.